apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2

```diff
@@ -1567,6 +1567,11 @@
         {% if console.ui.is_request_path_authentication_enabled is defined %}
         "isRequestPathAuthenticationEnabled": {{ console.ui.is_request_path_authentication_enabled }},
         {% endif %}
+        {% if console.ui.enableEmailCustomTemplate is defined %}
```

Comment:
```suggestion
        {% if console.ui.enable_email_custom_template is defined %}
```

`enableEmailCustomTemplate` should be in snake case, since that's the toml convention.
Unless we are rendering a dynamic object, lets follow the convension.

Reply:
```toml
        {% if console.ui.enable_custom_email_templates is defined %}
            "enableCustomEmailTemplates": {{ console.ui.enable_custom_email_templates }},
        {% else %}
            "enableCustomEmailTemplates": true,
        {% endif %}
```

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/4489/commits/8d5c986359ea5c2d65c78624aee725e0472996a6